FreeRTOS_DHCP.c

```diff
@@ -1162,7 +1162,7 @@
     static BaseType_t prvSendDHCPDiscover( void )
     {
         BaseType_t xResult = pdFAIL;
-        uint8_t const * pucUDPPayloadBuffer;
+        uint8_t * pucUDPPayloadBuffer;
```

Comment:
I am not sure why you want to make `pucUDPPayloadBuffer` a const pointer? A few lines further down there is this cast:
~~~c
void * pvCopyDest = &pucUDPPayloadBuffer[ dhcpFIRST_OPTION_BYTE_OFFSET + dhcpREQUESTED_IP_ADDRESS_OFFSET ];
~~~
and data will be written to the buffer.

Reply:
I think that the changes are all good, thank you.

Reply:
Hm yup, you can ignore my message too, removing const does seem better, and removes the warning I got. Protip, dont write (read?) code late at night. :-P

Reply:
@tomstorey, I was about to type us a reply saying that I had already removed the `const` (It is late here 🙂 - your theory about not reading code at night seems correct). But then I saw this message (by accident) and stopped. :)